production/helm/loki/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
 appVersion: 2.8.0
-version: 5.1.0
+version: 5.1.1
```

Comment:
With https://github.com/grafana/loki/commit/afee8bcabf5bba6c962eaad039eca4174a0a867a we are at `5.2.1` now.